management/src/main/java/org/ehcache/management/registry/DefaultManagementRegistryConfiguration.java

```diff
@@ -89,30 +102,26 @@ public String getCollectorExecutorAlias() {
   @Override
   public String toString() {
     return "DefaultManagementRegistryConfiguration{" + "context=" + context +
-        ", tags=" + tags +
-        ", collectorExecutorAlias='" + collectorExecutorAlias + '\'' +
-        '}';
+      ", tags=" + tags +
+      ", collectorExecutorAlias='" + collectorExecutorAlias + '\'' +
+      ", productInstanceId='" + productInstanceId + '\'' +
+      '}';
   }
 
   @Override
   public boolean equals(Object o) {
     if (this == o) return true;
     if (o == null || getClass() != o.getClass()) return false;
-
     DefaultManagementRegistryConfiguration that = (DefaultManagementRegistryConfiguration) o;
-
-    if (!tags.equals(that.tags)) return false;
-    if (!context.equals(that.context)) return false;
-    return collectorExecutorAlias != null ? collectorExecutorAlias.equals(that.collectorExecutorAlias) : that.collectorExecutorAlias == null;
-
+    return Objects.equals(tags, that.tags) &&
+      Objects.equals(productInstanceId, that.productInstanceId) &&
+      Objects.equals(context, that.context) &&
+      Objects.equals(collectorExecutorAlias, that.collectorExecutorAlias);
   }
 
   @Override
   public int hashCode() {
-    int result = tags.hashCode();
-    result = 31 * result + context.hashCode();
-    result = 31 * result + (collectorExecutorAlias != null ? collectorExecutorAlias.hashCode() : 0);
-    return result;
+    return Objects.hash(tags, productInstanceId, context, collectorExecutorAlias);
```

Comment:
The `productInstanceId` should be enough as an hashcode

Reply:
You know what, to me, having an equal/hashcode in a config class like this is useless... I wanted to keep the original behaviour but if you are fine, I would be pleased to completely remove them!

Reply:
I'm fine to remove them if no one is supposed to put these things in a HashMap. Useless code is useless code.

Reply:
Sorry actually this is used! used to verify that a config is equal to another one - so all attributes need to be there.